01week/datatypes.js

```diff
@@ -14,7 +14,8 @@ Null
 Undefined
 Number
 NaN
-String*/
+String
+I'm using an alert to show results*/
 alert(typeof 1) 
 //5.Write a JavaScript program that adds 2 numbers together.
 const c = a + b;
```

Comment:
This is a variable, I need a function that accepts two arguments and returns the sum.